<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myCustomer</class>
 <widget class="QMainWindow" name="myCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myCustomer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>651</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="customer_name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer's name</string>
    </property>
   </widget>
   <widget class="QLabel" name="customer_numberphone">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="customer_ID">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number phone</string>
    </property>
   </widget>
   <widget class="QLabel" name="customer_point">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Accumulation point</string>
    </property>
   </widget>
   <widget class="QLabel" name="customer_BOD">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Birth</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCusManager">
    <property name="title">
     <string>Customer Manager</string>
    </property>
   </widget>
   <addaction name="menuCusManager"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
